VideoLocker/res/layout/row_download_list.xml

```diff
@@ -41,6 +41,7 @@
 
         <ImageView
             android:id="@+id/close_btn"
+            android:contentDescription="@string/cancel_download"
```

Comment:
Download a video then go to the list of downloading videos. There is a "X" icon that a user can click on to stop the download. The contentDescription is now "Cancel download"